### DISCUSSION TITLE
How to use Lemlib for relative coordinates autonomous coding?

### DISCUSSION THREAD

**Discussion by Hazeline2018** [LemLib/LemLib#168] [Q&A] [answered]
Hello,

I used Lemlib to code my autonomous routines by using absolutely coordinates as shown in the path generator plot. However, the starting point of the robot varies for each game. So in order to accommodate all 4 quadrants, I had to make 4 images, one for each quadrant. This is a bit cumbersome and taking 4 of the 8 slots on the brain. I'm wondering if there is a way to code with relative coordinates. This way, I can only make 2 images to cover all 4 quadrants, if you know what I mean.
Thanks,
Terry

**Comment by nic-cheng** [comment, marked as answer]
You only need 2 quadrants (one for close side, one for far side) regardless of whether you're using absolute or relative positioning.

Being on blue or red alliance doesn't change the pathing at all.

**Reply by Hazeline2018** [reply]
you are absolutely right! I didn't get my ahead around it. :-) Thanks for the prompt answer!
